2024-12-08 08:26:08,623:DEBUG:certbot._internal.main:certbot version: 3.0.1
2024-12-08 08:26:08,623:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-12-08 08:26:08,623:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/var/www/certbot', '--email', 'tbongkim03@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'parkingissue.online']
2024-12-08 08:26:08,624:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-12-08 08:26:08,637:DEBUG:certbot._internal.log:Root logging level set at 30
2024-12-08 08:26:08,639:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-12-08 08:26:08,640:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A separate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7b7c3ba563c0>
Prep: True
2024-12-08 08:26:08,640:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7b7c3ba563c0> and installer None
2024-12-08 08:26:08,640:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-12-08 08:26:08,683:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2100863967', new_authzr_uri=None, terms_of_service=None), 9c1f5364519649d68a1e048dbf4abd52, Meta(creation_dt=datetime.datetime(2024, 12, 8, 8, 21, 33, tzinfo=<UTC>), creation_host='fc45881219b9', register_to_eff=None))>
2024-12-08 08:26:08,684:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-12-08 08:26:08,686:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-12-08 08:26:09,241:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 746
2024-12-08 08:26:09,242:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 08:26:09 GMT
Content-Type: application/json
Content-Length: 746
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "P0uZTCDaCsA": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-12-08 08:26:09,243:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for parkingissue.online
2024-12-08 08:26:09,254:DEBUG:acme.client:Requesting fresh nonce
2024-12-08 08:26:09,254:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-12-08 08:26:09,439:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-12-08 08:26:09,442:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 08:26:09 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: Uqfi4Yotvo3XaI2CPrOe3ajcaaqKuMga_2MzdWUW9cZf4ndWzLg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-12-08 08:26:09,443:DEBUG:acme.client:Storing nonce: Uqfi4Yotvo3XaI2CPrOe3ajcaaqKuMga_2MzdWUW9cZf4ndWzLg
2024-12-08 08:26:09,444:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "parkingissue.online"\n    }\n  ]\n}'
2024-12-08 08:26:09,453:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDg2Mzk2NyIsICJub25jZSI6ICJVcWZpNFlvdHZvM1hhSTJDUHJPZTNhamNhYXFLdU1nYV8yTXpkV1VXOWNaZjRuZFd6TGciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "TQLwBVZPmDBHtVPmbL_juKlDUFxOr_82i1WzPzIYZrjFO5RWDXbFJlBsQl6g6yw3i0KBt09pvUOj1HIcCdbz5uUjaA_Z5VvA5asIRFrRTPqrC8j0Ow1N3rq_m0YXgHlA3GQE3hMn7xcbqSdmK8Dlp8dAvcW8qmBdlqS7JIaghVAIhmATNxnuMm1zanBC6RPZfqDf2v0slCRh_K49O_uDSDj8PGjKCiqo3ZA384iIMJUpSELBiqfwG-atv4qTUJfexLb0I2yQp2iEq_lbdwWKYr2DG6EkgXm9-L-xQ8AfTOqsmstNb_42VOCP3t3lF6LVkcM9cc51zTbBHBNyvbyccw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInBhcmtpbmdpc3N1ZS5vbmxpbmUiCiAgICB9CiAgXQp9"
}
2024-12-08 08:26:09,802:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 353
2024-12-08 08:26:09,802:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 08 Dec 2024 08:26:09 GMT
Content-Type: application/json
Content-Length: 353
Connection: keep-alive
Boulder-Requester: 2100863967
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2100863967/331065456427
Replay-Nonce: Uqfi4YotYg3DKkTpPyWHEBe1e9yKVb1fPnWjdImmgKBsmZn4-qM
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-12-15T08:26:09Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "parkingissue.online"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2100863967/441679202927"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2100863967/331065456427"
}
2024-12-08 08:26:09,803:DEBUG:acme.client:Storing nonce: Uqfi4YotYg3DKkTpPyWHEBe1e9yKVb1fPnWjdImmgKBsmZn4-qM
2024-12-08 08:26:09,803:DEBUG:acme.client:JWS payload:
b''
2024-12-08 08:26:09,805:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2100863967/441679202927:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDg2Mzk2NyIsICJub25jZSI6ICJVcWZpNFlvdFlnM0RLa1RwUHlXSEVCZTFlOXlLVmIxZlBuV2pkSW1tZ0tCc21abjQtcU0iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMDA4NjM5NjcvNDQxNjc5MjAyOTI3In0",
  "signature": "GTnxD9rrtDMgQ7LzfqDWvapwpqhqh7y_rkydk8xntOqUlqM_OscJHsBQSD7DKPJGlb58tbmckUZFRQm24MLskdZP5d2fyV9-RNW99y7SxO2kfzh8CExTnDc2wx2nwsBW39kHSYyLXULQbN67OpT5hoPi7V54xa6esvpxbewr1dEebveqzlWEeJo0mI2ZC6fUhrEvY_AoncjHh9hHLqulgxkPkRPbPcD_rYc2Hla8MEQyERiY2neanFo2yUx_ZXlLQKfVNkDfD2kMBmCiNuCD5wKjMnz0eUs_bZuXY-02cIusF_0zK9V2RR2kNqGHvNMmahbWtMUd4SDKxAREEw0TUA",
  "payload": ""
}
2024-12-08 08:26:09,993:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2100863967/441679202927 HTTP/1.1" 200 827
2024-12-08 08:26:09,994:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 08:26:09 GMT
Content-Type: application/json
Content-Length: 827
Connection: keep-alive
Boulder-Requester: 2100863967
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: vyQW5O6tOXACXGYG6Z3ICmmXweK8KEyql9qZk4IN_cbQNoOwXCc
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "parkingissue.online"
  },
  "status": "pending",
  "expires": "2024-12-15T08:26:09Z",
  "challenges": [
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/HNjvzA",
      "status": "pending",
      "token": "YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/bFP3Ag",
      "status": "pending",
      "token": "YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o"
    },
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/d0exGw",
      "status": "pending",
      "token": "YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o"
    }
  ]
}
2024-12-08 08:26:09,994:DEBUG:acme.client:Storing nonce: vyQW5O6tOXACXGYG6Z3ICmmXweK8KEyql9qZk4IN_cbQNoOwXCc
2024-12-08 08:26:09,994:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-12-08 08:26:09,994:INFO:certbot._internal.auth_handler:http-01 challenge for parkingissue.online
2024-12-08 08:26:09,995:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2024-12-08 08:26:09,995:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2024-12-08 08:26:09,996:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o
2024-12-08 08:26:09,996:DEBUG:acme.client:JWS payload:
b'{}'
2024-12-08 08:26:09,997:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/bFP3Ag:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDg2Mzk2NyIsICJub25jZSI6ICJ2eVFXNU82dE9YQUNYR1lHNlozSUNtbVh3ZUs4S0V5cWw5cVprNElOX2NiUU5vT3dYQ2MiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIxMDA4NjM5NjcvNDQxNjc5MjAyOTI3L2JGUDNBZyJ9",
  "signature": "C6s0BctgCq-1sbbBIvIrRVk8dBWYco5eZ_ja6S602KLacT3vy5yPyhY_TaFqaxDSwpTFhIPAVU2qtlIWvgrEWTWooAKw3CAmiW0V25Q2pYtAJBBcLj7snY50kyVCiXg0vvTZPUsjjwF4wH6WdiMo_mKIRWD_tzv1ns06XmnfU_TLatQ9-PsBu7Xj20oufHOmP-w6CdHTgwkUyTqD5fPq06i7eY3sSE0yjDhCX7umtcQgA-nalQ3eF1No02AUSy61PjFDRjvfweLheP4-hndjce7AJ_yHduOt2U5vLsBGKrRVUmpBfm-ZmlmDDAQe5coakIf2e4Bh5Ht33V8qU2rY1A",
  "payload": "e30"
}
2024-12-08 08:26:10,184:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2100863967/441679202927/bFP3Ag HTTP/1.1" 200 195
2024-12-08 08:26:10,185:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 08:26:10 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2100863967
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2100863967/441679202927>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/bFP3Ag
Replay-Nonce: Uqfi4YotVDWdIQ-jNhCfkZP9gO7Kt78vGfuvnLV3G-cziZX5xyk
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/bFP3Ag",
  "status": "pending",
  "token": "YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o"
}
2024-12-08 08:26:10,185:DEBUG:acme.client:Storing nonce: Uqfi4YotVDWdIQ-jNhCfkZP9gO7Kt78vGfuvnLV3G-cziZX5xyk
2024-12-08 08:26:10,185:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-12-08 08:26:11,185:DEBUG:acme.client:JWS payload:
b''
2024-12-08 08:26:11,187:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2100863967/441679202927:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjEwMDg2Mzk2NyIsICJub25jZSI6ICJVcWZpNFlvdFZEV2RJUS1qTmhDZmtaUDlnTzdLdDc4dkdmdXZuTFYzRy1jemlaWDV4eWsiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMDA4NjM5NjcvNDQxNjc5MjAyOTI3In0",
  "signature": "PZRkY8O4LixqvzUgeI2kBvETwpKzSdohTyFnL-1JRjjbDBcH35_ZiO1CzBtXJPEmnIibkUmkgyiXccPuISmN90Ty4gJtxE-K6lMeGaHiU7itVH8AHs4TZT0VFUEW9rawdX8Q9df3_qXjv518HtkNIBDgX26f4ceNnH2a5eKa7GdvBfFyOqnAP1CEj1IHNAUrB7pgq1BF_Y97JbuNO9bzozL1ppUO7HxnXNT_37Jq1JIdjfPNVRE9L8LOMtT9J8Qw4CbWc0M6QPkL8NvRIQCId8RlS6MBfTjn_AOl2nkbLqNBDr6yBpdiX0pD_d7QGgZww8_jLZJ9_eUsLLoiti9pjg",
  "payload": ""
}
2024-12-08 08:26:11,374:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2100863967/441679202927 HTTP/1.1" 200 1046
2024-12-08 08:26:11,375:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 08 Dec 2024 08:26:11 GMT
Content-Type: application/json
Content-Length: 1046
Connection: keep-alive
Boulder-Requester: 2100863967
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: Uqfi4YotB1t9mFB4lJDY__fCXELlWh8XpAKL21hut0ugR6hnswQ
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "parkingissue.online"
  },
  "status": "invalid",
  "expires": "2024-12-15T08:26:09Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2100863967/441679202927/bFP3Ag",
      "status": "invalid",
      "validated": "2024-12-08T08:26:10Z",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "3.37.20.117: Fetching http://parkingissue.online/.well-known/acme-challenge/YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o: Connection refused",
        "status": 400
      },
      "token": "YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o",
      "validationRecord": [
        {
          "url": "http://parkingissue.online/.well-known/acme-challenge/YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o",
          "hostname": "parkingissue.online",
          "port": "80",
          "addressesResolved": [
            "3.37.20.117"
          ],
          "addressUsed": "3.37.20.117"
        }
      ]
    }
  ]
}
2024-12-08 08:26:11,375:DEBUG:acme.client:Storing nonce: Uqfi4YotB1t9mFB4lJDY__fCXELlWh8XpAKL21hut0ugR6hnswQ
2024-12-08 08:26:11,375:INFO:certbot._internal.auth_handler:Challenge failed for domain parkingissue.online
2024-12-08 08:26:11,375:INFO:certbot._internal.auth_handler:http-01 challenge for parkingissue.online
2024-12-08 08:26:11,375:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: parkingissue.online
  Type:   connection
  Detail: 3.37.20.117: Fetching http://parkingissue.online/.well-known/acme-challenge/YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-12-08 08:26:11,376:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-12-08 08:26:11,376:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-12-08 08:26:11,377:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-12-08 08:26:11,377:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/YQmDt-qnlvyOf43TaqBvJqAylAe1GUIHMIlF4Bl0J2o
2024-12-08 08:26:11,377:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-12-08 08:26:11,377:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1876, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1578, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 142, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-12-08 08:26:11,379:ERROR:certbot._internal.log:Some challenges have failed.
